Source/ZoomNet/Models/DashboardMeetingMetricsPaginationObject.cs

```diff
@@ -0,0 +1,59 @@
+using Newtonsoft.Json;
+using ZoomNet.Resources;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Pagination Object for meeting metrics returned as deep nested properties in some <see cref="IDashboards"/> endpoints.
+	/// </summary>
+	public class DashboardMeetingMetricsPaginationObject
```

Comment:
Is this class really necessary? I have a feeling we could simply use `PaginatedResponseWithTokenAndDateRange<DashboardMeetingMetrics>`

Reply:
The problem with reusing `PaginatedResponseWithTokenAndDateRange<DashboardMeetingMetrics>` is the json property name for the records collection.
Since the result is a nested object, I did not find a good hook to control the parsing, aside from creating a custom resolver class like you did for `Meeting`.
Of the two options, this felt easier to maintain and understand.

Reply:
Ok, i understand.